test/test_tree_3d_part2.jl

```diff
@@ -1,37 +1,51 @@
-module TestExamples3DPart1
+module TestExamplesTreeMesh3DPart2
 
 using Test
 using Trixi
 
 include("test_trixi.jl")
 
-# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
-EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_3d_dgsem")
-
 # Start with a clean environment: remove Trixi output directory if it exists
 outdir = "out"
 isdir(outdir) && rm(outdir, recursive=true)
 
-@testset "3D-Part1" begin
+@testset "TreeMesh3D Part 2" begin
 
 # Run basic tests
 @testset "Examples 3D" begin
   # Linear scalar advection
-  include("test_examples_3d_advection.jl")
+  include("test_tree_3d_advection.jl")
 
   # Hyperbolic diffusion
-  include("test_examples_3d_hypdiff.jl")
-
-  # Compressible Euler
-  include("test_examples_3d_euler.jl")
+  include("test_tree_3d_hypdiff.jl")
 
   # Compressible Euler with self-gravity
-  include("test_examples_3d_eulergravity.jl")
+  include("test_tree_3d_eulergravity.jl")
+end
+
+
+@trixi_testset "Additional tests in 3D" begin
```

Comment:
Additional tests seem to be somewhat scattered around, sometimes they are in the `part1` file, sometimes in `part2`. Would it make sense - and is it even doable - to always include them in the same file? Or, e.g., collect them in a file `test_tree_3d_additional.jl` that can then be included in any of the main files, such that it is clear where to go when adding a new small-scale test?

This is not a show stopper; if you feel the effort isn't worth the outcome, feel free to let me know and close the conversation.

Reply:
I moved them around while trying to distribute the CI times more uniformly. Thus, putting them always in the same `part` is not easily possible (unless we want to also allow splitting the `euler` tests in a single dimension, for example).

Reply:
I'm sure it's ok as it is, I was just wondering. Should we find that over time there are too many tests scattered around and nobody knows where to look anymore, we can revisit.